src/main/java/com/example/icecreamup/IcecreamupController.java

```diff
@@ -40,7 +38,7 @@ public String read() {
 		return "start";
 	}
 	
-	@RequestMapping("/")
+	@RequestMapping(value="/", method=RequestMethod.GET)
```

Comment:
Spring should now map to the "/" endpoint.  This means that if  someone types in the base URL, they will be redirected to the "/start" page.